src/main/java/com/redhat/devtools/lsp4ij/usages/LSPUsageElementDescriptionProvider.java

```diff
@@ -53,6 +54,11 @@ public String getElementDescription(@NotNull PsiElement element, @NotNull Elemen
             }
         }
 
+        // If it's not one of ours, don't provide a description
+        if (!(element instanceof LSPPsiElement)) {

```

Comment:
Should be pretty self-explanatory, but if we get an element that's not ours, don't return anything here.